numpy/ma/core.pyi

```diff
@@ -1113,9 +1113,9 @@ def putmask(a, mask, values): ...
 def transpose(a, axes=...): ...
 def reshape(a, new_shape, order=...): ...
 def resize(x, new_shape): ...
-def ndim(obj): ...
+def ndim(obj: ArrayLike) -> int: ...
 def shape(obj): ...
```

Comment:
`np.shape` has 7 overloads? looks like complicated, i'll try that separately so as to not block the other simpler ones i'm adding here

Reply:
Yea, shape-typing is no walk in the park 😅. But let's indeed non overcomplicate things, and keep shape-typing out of `ma` for now.